static/js/message_edit.js

```diff
@@ -242,6 +242,21 @@ function edit_message (row, raw_content) {
             }
         }, 1000);
     }
+    $( "#message_edit_topic" ).focus(function () {
+        if (original_topic === "(no topic)") {
+            $( this ).val("");
+            $(this).attr("placeholder", "(no topic)");
+        }
+    });
+    $( "#message_edit_topic" ).click(function () {
+        if (original_topic === "(no topic)") {
+            this.placeholder = "(no topic)";
+            this.value = "";
+        } else {
+            $( this ).focus();
+            $( this ).select();
+        }
+    });
```

Comment:
Cool!  A quick style thing is that in Zulip, we always write `$(this)`, not `$( this )` (for all function calls); can you clean that up?

Also, if you search for `message_edit` in static/js/click_handlers.js, I think you might be able to simplify this code by changing the existing click handlers there (rather than adding another handler here)...